python/glow/gwas/log_reg.py

```diff
@@ -69,6 +71,10 @@ def logistic_regression(
         A Spark DataFrame that contains
 
         - All columns from ``genotype_df`` except the ``values_column`` and the ``genotypes`` column if one exists
+        - ``effect``: The effect size (if approximate Firth correction was applied)
+        - ``stderror``: Standard error of the effect size (if approximate Firth correction was applied)
+        - ``correction``: The correction method used (if the correction test method is not ``none``).
```

Comment:
Hm, I'm not sure about this semantic. It should be straightforward to know if there was a fitting error. Maybe it would be more user friendly to have a `fit_status` field? Where if positive the status indicates the number of iterations to fit, `-1` means no convergence, `-2` means exploded. Or even a boolean that indicates if the correction succeeded.